File: Core/Models/Price.cs
using System;

namespace PriceAdvisor.Core.Models
{
    public class Price
    {
        public int Id { get; set; }
        public string Value { get; set; }
        public DateTime UpdatedAt { get; set; }
        public int EshopId { get; set; }
        public EShop EShop {get;set;}
        public bool Edited {get;set;}
        public int ProductId { get; set; }
        public Product Product {get;set;}
    }
}

File: ClientApp/app/models/price.ts
export interface Price {
    id: number; 
    value: number;
    updatedAt: string | null;
    eshopId: number;  
  }
  
    export interface SavePrice {
    id: number; 
    value: number;   
    eshopId: number; 
    updatedAt: string | null;
    productId: number;   
    edited: boolean;
  }